docs/Learning-Environment-Design.md

```diff
@@ -83,6 +83,12 @@ training a maze-solving agent, you would probably want to change the maze itself
 for each training episode. Otherwise, the agent would probably on learn to solve
 one, particular maze, not mazes in general.
 
+#### Academy Pitfalls
+
+In some games, it may be desirable to step the environment with the Academy manually via `Academy.Instance.EnvironmentStep()`.
```

Comment:
Strange that there is not a section about `Academy.Instance.EnvironmentStep()` already in this doc (out of scope for this PR). Stepping the Academy manually is an advanced scenario, maybe call this out?